pkgs/development/compilers/jetbrains-jdk/jcef.nix

```diff
@@ -306,5 +303,9 @@ stdenv.mkDerivation rec {
     description = "Jetbrains' fork of JCEF";
     license = lib.licenses.bsd3;
     homepage = "https://github.com/JetBrains/JCEF";
+    platforms = [
+      "aarch64-linux"
+      "x86_64-linux"
+    ];
```

Comment:
Why these specific platforms and not the full `lib.platforms.linux` list?

The previous assertion was just "not darwin" which would imply all linux platforms are supported.